Essentials/src/messages.properties

```diff
@@ -592,6 +593,7 @@ ptimeCommandDescription=Adjust player's client time. Add @ prefix to fix.
 ptimeCommandUsage=/<command> [list|reset|day|night|dawn|17:30|4pm|4000ticks] [player|*]
 pweatherCommandDescription=Adjust a player's weather
 pweatherCommandUsage=/<command> [list|reset|storm|sun|clear] [player|*]
+priceChanged=\u00a76The price of \u00a7c{0}x {1}\u00a76 has changed from \u00a7e{2}\u00a76 to \u00a7e{3}\u00a76.
```

Comment:
We should prompt the player to right-click the sign again if they want to buy/sell at the new price.

Reply:
Done